Jenkinsfile.deploy

```diff
@@ -255,14 +266,16 @@ pipeline {
 
 void updateQuickstartsVersions(){
     maven.mvnSetVersionProperty(getOptaplannerQuickstartsMavenCommand(), "version.org.optaplanner", getProjectVersion())
-    maven.mvnVersionsUpdateParentAndChildModules(getOptaplannerQuickstartsMavenCommand(), getProjectVersion())
+    maven.mvnVersionsUpdateParentAndChildModules(getOptaplannerQuickstartsMavenCommand(), getProjectVersion(), !isRelease())
     gradleVersionsUpdate(quickstartsRepository, getProjectVersion())
 
-    dir(quickstartsRepository) {
-        assert !sh (script:
-                'grep -Rn "SNAPSHOT" --include={pom.xml,build.gradle} | ' +
-                'grep -v -e "0.1.0-SNAPSHOT" -e ">1.0-SNAPSHOT" | ' +
-                'cat', returnStdout: true)
+    if (isRelease()) {
+        dir(quickstartsRepository) {
+            assert !sh (script:
+                    'grep -Rn "SNAPSHOT" --include={pom.xml,build.gradle} | ' +
+                    'grep -v -e "0.1.0-SNAPSHOT" -e ">1.0-SNAPSHOT" | ' +
+                    'cat', returnStdout: true)
+        }
     }
```

Comment:
We can additionally check if there is something other SNAPSHOT left except new project version snapshot
if(isCreatePr()){
dir(quickstartsRepository){
assert !sh (script:
                    'grep -Rn "SNAPSHOT" --include={pom.xml,build.gradle} | ' +
                    'grep -v -e "${getProjectVersion()}" | ' +
                    'cat', returnStdout: true)
}
}

Reply:
> We can additionally check if there is something other SNAPSHOT left except new project version snapshot
> if(isCreatePr()){
> dir(quickstartsRepository){
> assert !sh (script:
> 'grep -Rn "SNAPSHOT" --include={pom.xml,build.gradle} | ' +
> 'grep -v -e "${getProjectVersion()}" | ' +
> 'cat', returnStdout: true)
> }
> }

DONE